modules/custom/az_carousel/config/install/field.field.node.az_carousel_item.field_az_media_image.yml

```diff
@@ -10,7 +10,7 @@ field_name: field_az_media_image
 entity_type: node
 bundle: az_carousel_item
 label: Image
-description: 'Minimum dimensions: 1400px (wide) x 500px (tall)'
+description: 'Minimum dimensions: 1110px (wide) x 500px (tall)'
```

Comment:
Should the wording be updated to be consistent with the way Person content type words it?
![image](https://github.com/az-digital/az_quickstart/assets/4400438/73ef390c-a5a8-4618-bfae-c94514781313)

Reply:
Created a followup issue to address this inconsistency: #3343.

We decided in our 2024-04-10 PR meeting that it'd be better to update az_person to use the az_carousel help text conventions than the other way around.